connectors/sources/tests/test_mysql.py

```diff
@@ -138,15 +146,15 @@ async def test_close_with_connection_pool():
     """Test close method of MySql with connection pool"""
     # Setup
     source = create_source(MySqlDataSource)
-
-    source.connection_pool = Connection
```

Comment:
Since we are not using the connection class now, we can remove `close()` and `wait_closed()` from the connection class.